src/Pages/ProductList/ProductListMain/MainHeader/HeaderOption/HeaderOption.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+export default class HeaderOption extends Component {
+  render() {
+    const {
+      data: { title, ActiveHeaderModal },
+      headerOptionClickedAt,
+      index,
+      clickHandlerBtnClicked,
+    } = this.props;
+    const isPushed = headerOptionClickedAt === index;
+    return (
+      <div>
+        <HeaderOptionTitle
+          isPushed={isPushed}
+          onClick={() => clickHandlerBtnClicked("headerOption", index)}
+        >
+          {title}
+        </HeaderOptionTitle>
+        <div>
+          {headerOptionClickedAt === index && headerOptionClickedAt !== 4 && (
+            <ActiveHeaderModal />
```

Comment:
```suggestion
            <HeaderModal />
```
HeaderModal은 그것대로 존재하고 이 내용이 보이면 active 되는거겠죠. 이름 자체가 active라는 상태 값을 가지고 있을 필요는 없습니다

Reply:
수정완료!